documents/BUILDINGS.md

```diff
@@ -0,0 +1,5 @@
+# Buildings
+
+[Reference Document](https://docs.google.com/spreadsheets/d/1H_UoOrvQGwsS0GIzRcd_AlPuNHbIsHRAz4Llo7vYrSU/edit?usp=sharing)
```

Comment:
No, we should not be sharing a random public google sheets. Either it's here or it's private.